core/src/main/java/org/jdbi/v3/core/result/ResultIterable.java

```diff
@@ -89,27 +89,79 @@ default void forEach(Consumer<? super T> action) {
     }
 
     /**
-     * Get the only row in the result set.
-     * @throws IllegalStateException if zero or multiple rows are returned
-     * @return the object mapped from the singular row in the results
+     * Returns the only row in the result set. Returns {@code null} if the row itself is
+     * {@code null}.
+     * @throws IllegalStateException if the result set contains zero or multiple rows
+     * @return the only row in the result set.
      */
-    default T findOnly() {
+    default T one() {
         try (ResultIterator<T> iter = iterator()) {
             if (!iter.hasNext()) {
-                throw new IllegalStateException("No element found in 'only'");
```

Comment:
In theory this might be a `NoSuchElementException`

Reply:
I guess we probably don't want to change that.

Reply:
I left it as `IllegalStateException` since we had several tests expecting that exception type.